services/ui-src/src/routes/ReviewSubmit/ReviewSubmit.tsx

```diff
@@ -1,29 +1,215 @@
+import { useContext, useEffect } from "react";
 // components
-import { Box, Heading, Text } from "@chakra-ui/react";
-import { ReportPage } from "components";
-import verbiage from "verbiage/pages/mcpar/mcpar-review-and-submit";
+import {
+  Box,
+  Button,
+  Flex,
+  Image,
+  Heading,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { Icon, Modal, ReportContext, ReportPage, Sidebar } from "components";
+// types
+import { ReportStatus, UserRoles } from "types";
+// utils
+import { useUser } from "utils";
+// form data
+import reviewVerbiage from "verbiage/pages/mcpar/mcpar-review-and-submit";
+import successVerbiage from "verbiage/pages/mcpar/mcpar-successful-submit";
+// assets
+import checkIcon from "assets/icons/icon_check_circle.png";
 
 export const ReviewSubmit = () => {
-  const { intro } = verbiage;
+  const { reportStatus, fetchReportStatus, updateReportStatus } =
+    useContext(ReportContext);
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  // get user's state
+  const { user } = useUser();
+  const { state, userRole } = user ?? {};
+  const reportYear = "2022";
+  const reportKey = `${state}${reportYear}`;
+
+  // TODO: get real program name per report
+  const programName = "tempName";
+
+  const reportDetails = {
+    key: reportKey,
+    programName: programName,
+  };
+
+  useEffect(() => {
+    fetchReportStatus(reportDetails);
+  }, []);
+
+  const submitForm = () => {
+    // TODO: Add check to make sure user filled out the form
+    if (userRole === UserRoles.STATE_USER || userRole === UserRoles.STATE_REP) {
+      updateReportStatus(reportDetails, ReportStatus.COMPLETED);
+    }
+    // TODO: Handle Error for other users coming on to the page
```

Comment:
Other users will be able to see the state of the form but not click the submit button, so this will likely just be a `disabled` button state instead of a check here. Can remove TODO.